vm-builder/files/set-disk-quota.sh

```diff
@@ -18,6 +18,7 @@ echo "Setting disk quota for $mountpoint to $size KB"
 # setquota -P <project_id> <block-softlimit> <block-hardlimit> <inode-softlimit> <inode-hardlimit> <filesystem>
 /neonvm/bin/setquota -P 0 0 "$size" 0 0 "$mountpoint"
 
-echo "DONE! Disk quota set"
-/neonvm/bin/repquota -P -a
+# Don't print the quota report, it's too verbose
+# echo "DONE! Disk quota set"
+# /neonvm/bin/repquota -P -a
```

Comment:
Maybe let's just remove, rather than commenting out?

Reply:
I think it's useful as info, to remind how to look at the quotas (repquota flags are hard to remember). So I'd keep it.